spec/requests/api/v0/kafkadata_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require 'ostruct'
+
+RSpec.describe Api::V0::EventsController::KafkaData do
+  let(:user_id) { 'd8388680-80cf-4fdb-95bb-829a46342115' }
+  let(:user_data) { { 'user_uuid': user_id } }
+  let(:session_uuid) { 'ed3cdd93-6688-4fcb-b5b3-da3e71052454' }
+  let(:compact_session_uuid) { CompactUuid.pack(session_uuid) }
+  let(:client_clock_occurred_at) { '2020-10-06T18:14:22Z' }
+  let(:client_clock_sent_at) { '2020-10-06T18:14:22Z' }
+  let(:received_at) { Time.parse('2020-10-06T18:14:27Z') }
+  let(:occured_at) do
+    TimeUtil.infer_actual_occurred_at_from_client_timestamps(
+      request_received_at: received_at,
+      client_clock_occurred_at: client_clock_occurred_at,
+      client_clock_sent_at: client_clock_sent_at
+    ).to_i
+  end
+
+  let(:data) do
```

Comment:
can we call this `api_data` to make clear for sure that it isn't the "kafka data"?

Reply:
done